DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/GroupDelegates/GroupDelegatesViewModelTests.cs

```diff
@@ -16,17 +14,24 @@ public class GroupDelegatesViewModelTests
         private readonly DelegateGroupsSideNavViewModel expectedNavViewModel =
             new DelegateGroupsSideNavViewModel(1, "Group name", DelegateGroupPage.Delegates);
 
-        private GroupDelegate[] groupDelegates = null!;
-
-        [SetUp]
-        public void SetUp()
+        private readonly GroupDelegate[] groupDelegates =
         {
-            groupDelegates = Builder<GroupDelegate>.CreateListOfSize(15)
-                .All()
-                .With(g => g.LastName = "Surname")
-                .With((g, i) => g.FirstName = NBuilderAlphabeticalPropertyNamingHelper.IndexToAlphabeticalString(i))
-                .Build().ToArray();
-        }
+            new GroupDelegate { FirstName = "A", LastName = "Surname" },
+            new GroupDelegate { FirstName = "B", LastName = "Surname" },
+            new GroupDelegate { FirstName = "C", LastName = "Surname" },
+            new GroupDelegate { FirstName = "D", LastName = "Surname" },
+            new GroupDelegate { FirstName = "E", LastName = "Surname" },
+            new GroupDelegate { FirstName = "F", LastName = "Surname" },
+            new GroupDelegate { FirstName = "G", LastName = "Surname" },
+            new GroupDelegate { FirstName = "H", LastName = "Surname" },
+            new GroupDelegate { FirstName = "I", LastName = "Surname" },
+            new GroupDelegate { FirstName = "J", LastName = "Surname" },
+            new GroupDelegate { FirstName = "K", LastName = "Surname" },
+            new GroupDelegate { FirstName = "L", LastName = "Surname" },
+            new GroupDelegate { FirstName = "M", LastName = "Surname" },
+            new GroupDelegate { FirstName = "N", LastName = "Surname" },
+            new GroupDelegate { FirstName = "O", LastName = "Surname" },
+        };

```

Comment:
Do you know what was causing the inconsistent passes? It looks like it could only be ordering, in which case you could fix it with an OrderBy() I'd have thought.

Reply:
We're specifically testing that the order by and paginate in the view model constructor works here. It's like it was generating the list incorrectly or something, so when we do the alphabetical ordering it didn't reach the expected point.

It didn't do it when debugging or running just a few tests and was intermittently happening for the full test run (It passed before I pushed, failed in the github build twice, passed in the jenkins build twice, failed when I re-ran locally to figure out what was wrong, but passed when I debugged them)

Reply:
Okay, could you just try ordering the NBuilder list by first name before the `ToArray()`? Then you can stick `[Repeat(100)]` on one of the tests which was failing intermittently to see if any of the runs fails.

Reply:
Crazy. Cool, let's just stick with this then.